rebar.config

```diff
@@ -39,7 +39,9 @@
 {deps, [
     {b64fast, {git, "https://github.com/ArweaveTeam/b64fast.git", {ref, "58f0502e49bf73b29d95c6d02460d1fb8d2a5273"}}},
     {jiffy, {git, "https://github.com/ArweaveTeam/jiffy.git", {ref, "74c956defa9116c85d76f77c3e9b5bd6de7bd39a"}}},
-    {cowboy, {git, "https://github.com/ninenines/cowboy", {tag, "2.12.0"}}}
+    {cowboy, {git, "https://github.com/ninenines/cowboy", {tag, "2.12.0"}}},
+    {gun, "2.0.1"},
```

Comment:
To double check here, do we need `gun` and `gproc` here?

Reply:
When building on my Mac I seemed to need it at least. It was not there before so I recon someone building it elsewhere maybe didn't need it. That being said I am new to Erlang so wouldn't know for certain.